src/anomalib/utils/sweep/helpers/callbacks.py

```diff
@@ -7,7 +7,7 @@
 from lightning.pytorch import Callback
 from omegaconf import DictConfig, ListConfig
 
-from anomalib.utils.callbacks import MetricsConfigurationCallback, PostProcessingConfigurationCallback
+from anomalib.utils.callbacks import MetricsConfigurationCallback
```

Comment:
Are we going to keep this callback?

Reply:
No. I have removed this in the follow up PR